superset-frontend/src/dashboard/components/Header.jsx

```diff
@@ -17,6 +17,8 @@
  * under the License.
  */
 /* eslint-env browser */
+/* eslint-disable camelcase */
```

Comment:
since this only affects one line, maybe you can limit the disable to only that line too by moving the comment?

Reply:
agree to use line-level disable rule.